config/rwahs/components/providence

```diff
@@ -28,7 +28,9 @@ function postDeploy {
     localSettingsRoot="${2}"
 
     ln -s "${localSettingsRoot}/setup.php" "${targetRoot}/${targetParent}/current/setup.php"
-    ln -s "${localSettingsRoot}/media/collectiveaccess" "${targetRoot}/${targetParent}/current/media/collectiveaccess"
+    ln -s "${localSettingsRoot}/../storage/media/collectiveaccess" "${targetRoot}/${targetParent}/current/media/collectiveaccess"
```

Comment:
Using a relative path here rather than adding a new parameter to the postDeploy hook